### PR TITLE
Update enable_i2c.md

### PR DESCRIPTION
**Pull request by webeling67**
More documentation on OS level for I2C.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @webeling67, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#17005).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Comment by ikifar2012** [comment]
Remember to reference your issue #16997

**Comment by webeling67** [comment]
> 
> 
> Remember to reference your issue #16997

Do I need to do something to get the pull request into the online?

File: source/_includes/common-tasks/enable_i2c.md [hunk 1]
```diff
@@ -1,23 +1,25 @@
 ## Enable I2C
 
-Home Assistant using the Home Assistant Operating System is a managed environment, which means you can't use existing methods to enable the I2C bus on a Raspberry Pi.
+Home Assistant using the Home Assistant Operating System is a managed environment, which means you can't use existing methods to enable the I2C bus on a Raspberry Pi. In order to enable I2C devices you will have to 
+- Enable I2C for the Home Assistant Operating System 
+- Enable Raspberry Hats including the I2C integration
+- Setup I2C sensors
 
-### Step by step instructions
+### Step by step instructions to enable I2C for the Home Assistant Operating System
```
Comment (frenck) [inline source/_includes/common-tasks/enable_i2c.md]: This page is about the Home Assistant Operating System there is no need to put that in a big header again. Please revert this line change.

File: source/_includes/common-tasks/enable_i2c.md [hunk 2]
```diff
@@ -62,3 +62,28 @@ You can enable i2c via this terminal:
...
+  - platform: raspihats
+
+# Enable BMP 280 sensor
+sensor:
+  - platform: bmp280
+    i2c_address: 0x76
+```
+
+After a reboot of host, you can find the sensor listed in `Developer Tools` in the tab `States`as `sensor.bmp280_temperature`and `sensor.bmp280_pressure`. 
+
```
Comment (frenck) [inline source/_includes/common-tasks/enable_i2c.md]: This is not related to this page. Please remove that. This belongs on a integration page. Not in the common tasks.

File: source/_includes/common-tasks/enable_i2c.md [hunk 3]
```diff
@@ -62,3 +62,28 @@ You can enable i2c via this terminal:
   sync
   reboot
   ```
+#### Check I2C activation on OS level
+
+After the reboot of the host, the I2C activation should now be visible under /dev. You may check by using the add-on `Terminal & SSH` and list the directory `/dev`. If `ls` shows an entry `i2c-1` or similar, your I2C config might work. You can check the status of I2C kernel modules by entering `lsmod | grep i2c`. If they are loaded, you should find at least the entry `i2c_dev`. Active usage of the modules is indicated by a number, e.g. `i2c_dev 20480 2` would indicate two active I2C data sources (e.g. a BMP280 temperature & pressure sensor) 
+
+An active I2C can also be check with a multi meter showing 3.3 V on the I2C pins GPIO2 and GPIO3.
```
Comment (frenck) [inline source/_includes/common-tasks/enable_i2c.md]: This section does not apply to a regular installation and requires third party add-ons. It should therefore not be added here.

File: source/_includes/common-tasks/enable_i2c.md [hunk 4]
```diff
@@ -32,19 +34,17 @@ and make sure the first partition is available.
...
+#### Use SD Card/Step 3 - Start with the new OS configuration
 
 - Insert the SD card back into your Raspberry Pi.
 - On startup, the `hassos-config.service` will automatically pickup the new
   `rpi-i2c.conf` configuration.
 - Another reboot might be necessary to make sure the just imported `rpi-i2c.conf` is
   present at boot time.
 
-The I2C devices should now be present under /dev.
+### Alternative to SD Card via Home Assistant Operating System Terminal
```
Comment (frenck) [inline source/_includes/common-tasks/enable_i2c.md]: Could you explain why this title needed changing?

File: source/_includes/common-tasks/enable_i2c.md [hunk 5]
```diff
@@ -1,23 +1,25 @@
...
+Home Assistant using the Home Assistant Operating System is a managed environment, which means you can't use existing methods to enable the I2C bus on a Raspberry Pi. In order to enable I2C devices you will have to 
+- Enable I2C for the Home Assistant Operating System 
+- Enable Raspberry Hats including the I2C integration
+- Setup I2C sensors
 
-### Step by step instructions
+### Step by step instructions to enable I2C for the Home Assistant Operating System
 
-You will need:
+#### Use SD Card/Step 1 - Access the Home Assistant Operating System boot partition
```
Comment (frenck) [inline source/_includes/common-tasks/enable_i2c.md]: This replacing of the header, removed the introduction paragraph.

File: source/_includes/common-tasks/enable_i2c.md [hunk 6]
```diff
@@ -1,23 +1,25 @@
...
-#### Step 1 - Access the Home Assistant Operating System boot partition
-
 Shutdown/turn-off your Home Assistant installation and unplug the SD card.
 Plug the SD card into an SD card reader and find a drive/file system named
 `hassos-boot`. The file system might be shown/mounted automatically. If not,
 use your operating systems disk management utility to find the SD card reader
 and make sure the first partition is available.
 
-#### Step 2 - Add files to enable I2C
+#### Use SD Card/Step 2 - Add files to enable I2C
```
Comment (frenck) [inline source/_includes/common-tasks/enable_i2c.md]: Let's not add "Use SD Card" to each header, instead, make the introduction correct.

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by webeling67** [comment]
Thanks for the thorough review. Reading through the forums and integration pages, I thought it would be helpful to have more details regarding a confirmation that I2C is actually active. I'll rework the titles.
I'll pull the BMP280 references.

File: source/_includes/common-tasks/enable_i2c.md (hunk 2 above)
Comment (webeling67) [reply source/_includes/common-tasks/enable_i2c.md]: Removed the section as requested

File: source/_includes/common-tasks/enable_i2c.md (hunk 3 above)
Comment (webeling67) [reply source/_includes/common-tasks/enable_i2c.md]: Removed the reference to third party add on. Kept lsmod and more explicit description of the dev folder

File: source/_includes/common-tasks/enable_i2c.md (hunk 4 above)
Comment (webeling67) [reply source/_includes/common-tasks/enable_i2c.md]: Shortend the section title

File: source/_includes/common-tasks/enable_i2c.md (hunk 5 above)
Comment (webeling67) [reply source/_includes/common-tasks/enable_i2c.md]: Section titles cleaned of SD card reference

File: source/_includes/common-tasks/enable_i2c.md (hunk 6 above)
Comment (webeling67) [reply source/_includes/common-tasks/enable_i2c.md]: Removed SD card reference

**Comment by webeling67** [comment]
Hi Franck , I am not 100% sure that the I2C needs activation of the raspihats to allow for I2C sensors to work. On my RPi3b I needed to get raspihats integration working to have a functional I2C sensor. Can you comment, if the initial section in the I2C paragraph

_

> In order to use I2C devices you will have to 
>  - Enable I2C for the Home Assistant Operating System 
>  - Configure the Raspberry Hats binary sensor integration to enable the I2C integration
>  - Setup I2C devices e.g. sensors

_ 

is actually correct and is also applicable to non Raspi plattforms, I would think that this is incorrect. However there are very frustrated discussions in the forum regarding I2C and Raspi and the lack of a "noob proof" documentation. As I share this view after trying I2C on RasPi, I started the documentation change.

File: source/_includes/common-tasks/enable_i2c.md [hunk 7]
```diff
@@ -1,23 +1,25 @@
 ## Enable I2C
 
-Home Assistant using the Home Assistant Operating System is a managed environment, which means you can't use existing methods to enable the I2C bus on a Raspberry Pi.
+Home Assistant using the Home Assistant Operating System which is a managed environment, which means you can't use existing methods to enable the I2C bus on a Raspberry Pi. In order to use I2C devices you will have to 
+- Enable I2C for the Home Assistant Operating System 
+- Configure the Raspberry Hats binary sensor integration to enable the I2C integration
+- Setup I2C devices e.g. sensors
 
-### Step by step instructions
+### Step by step instructions to enable I2C via SD card
```
Comment (agners) [inline source/_includes/common-tasks/enable_i2c.md]:
```suggestion
### Enable I2C with an SD card reader
```

File: source/_includes/common-tasks/enable_i2c.md [hunk 8]
```diff
@@ -32,17 +34,15 @@ and make sure the first partition is available.
...
 - Insert the SD card back into your Raspberry Pi.
 - On startup, the `hassos-config.service` will automatically pickup the new
   `rpi-i2c.conf` configuration.
 - Another reboot might be necessary to make sure the just imported `rpi-i2c.conf` is
   present at boot time.
 
-The I2C devices should now be present under /dev.
-
-### From Home Assistant Operating System Terminal
+### Via Home Assistant Operating System Terminal
```
Comment (agners) [inline source/_includes/common-tasks/enable_i2c.md]:
```suggestion
### Enable I2C via Home Assistant Operating System Terminal
```

File: source/_includes/common-tasks/enable_i2c.md [hunk 9]
```diff
@@ -62,3 +62,9 @@ You can enable i2c via this terminal:
   sync
   reboot
   ```
+### Check I2C activation on OS level
+
+After the reboot of the host, the I2C activation should now be visible under /dev.  If the folder contains an entry `i2c-1` or similar, your I2C config might work. You can check the status of I2C kernel modules by entering `lsmod | grep i2c`. If they are loaded, you should find at least the entry `i2c_dev`. Active usage of the modules is indicated by a number, e.g. `i2c_dev 20480 2` would indicate two active I2C data sources (e.g. a BMP280 temperature & pressure sensor)
```
Comment (agners) [inline source/_includes/common-tasks/enable_i2c.md]:
```suggestion
After rebooting the host there should be `i2c-0` and similar device files in `/dev`. If such device files are missing, enabling I2C failed for some reason. You can check the status of I2C kernel modules by using `lsmod | grep i2c` in the terminal. If they are loaded, you should find at least the entry `i2c_dev`. Active usage of the modules is indicated by a number, e.g. `i2c_dev 20480 2` would indicate two active I2C device files.
```

File: source/_includes/common-tasks/enable_i2c.md [hunk 10]
```diff
@@ -62,3 +62,9 @@ You can enable i2c via this terminal:
   sync
   reboot
   ```
+### Check I2C activation on OS level
```
Comment (agners) [inline source/_includes/common-tasks/enable_i2c.md]:
```suggestion
### Troubleshooting
```

File: source/_includes/common-tasks/enable_i2c.md [hunk 11]
```diff
@@ -1,23 +1,25 @@
 ## Enable I2C
 
-Home Assistant using the Home Assistant Operating System is a managed environment, which means you can't use existing methods to enable the I2C bus on a Raspberry Pi.
+Home Assistant using the Home Assistant Operating System which is a managed environment, which means you can't use existing methods to enable the I2C bus on a Raspberry Pi. In order to use I2C devices you will have to 
+- Enable I2C for the Home Assistant Operating System 
+- Configure the Raspberry Hats binary sensor integration to enable the I2C integration
```
Comment (agners) [inline source/_includes/common-tasks/enable_i2c.md]: This is only required if you use the I2C hat. I2C can also be used without that hat, so I would just drop this line

```suggestion
```

**Review by agners — DISMISSED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @webeling67 👍 

And thanks @agners for confirming/tweak.

This looks good to go now 🎉
